shell/platform/android/ndk_helpers.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_ANDROID_NDK_HELPERS_H_
+#define FLUTTER_SHELL_PLATFORM_ANDROID_NDK_HELPERS_H_
+
+#include "flutter/fml/platform/android/jni_util.h"
+
+#include "flutter/impeller/renderer/backend/gles/gles.h"
+
+#include <android/hardware_buffer.h>
+
+namespace flutter {
+
+// A collection of NDK functions that are available depending on the version of
+// the Android SDK we are linked with at runtime.
+class NDKHelpers {
+ public:
+  // REQUIRED: Must be called before any methods below are.
+  static void Init();
+
+  // API Version 26
+  static bool HardwareBufferSupported();
+  static AHardwareBuffer* AHardwareBuffer_fromHardwareBuffer(
```

Comment:
Here and elsewhere, please prefer PascalCase. So `AHardwareBufferFromJavaHardwareBuffer`.

Reply:
The names are coming directly from the NDK which will make searching for usage simple. Let me know if you feel strongly about rewriting these names here.

Reply:
No. No strong opinions. Seem like a good reason to break convention.